openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/validators/custom-date-validators.ts

```diff
@@ -9,6 +9,7 @@ export class CustomDateValidator {
                 const inputDate = new Date(field.value);
                 if (!isNaN(inputDate.getTime())) {
                     const minDate = this._resolveTime(minTime);
+                    minDate.setDate(minDate.getDate() - 1);
                     if (inputDate < minDate) {
```

Comment:
Couldn't you just do this?
```suggestion
                    if (inputDate <= minDate) {
```

Reply:
This is not working as well as inputDate is having a time and min Date is having time which is different. So the date comparing is showing a different result.

Reply:
That makes sense. I think Kevin's idea to set the time on min date would work best